graylog2-web-interface/src/components/common/ColorPickerPopover.tsx

```diff
@@ -29,6 +29,17 @@ const StyledPopover = styled(Popover)`
   }
 `;
 
+type Props = {
+  id: string,
+  placement: 'top' | 'right' | 'bottom' | 'left',
```

Comment:
Since it is not required in the PropTypes, should this have a `?` proceeding it?
```suggestion
  placement?: 'top' | 'right' | 'bottom' | 'left',
```